src/client/components/SignupForm/index.js

```diff
@@ -0,0 +1,22 @@
+const React = require('react');
+
+const SignupForm = (props) => (
+  <form required className="signup-form" method="POST" action="/api/signup"
+    onSubmit={props.onSubmit}>
+    <input type="text" className="textInput" name="username"
```

Comment:
please be consequent , use dash in class names or camel case, but dont mix them. Discuss this with your team members.